Makefile

```diff
@@ -76,6 +76,8 @@ else
 start: update
 	CONTIV_NODE_OS=${CONTIV_NODE_OS} vagrant up
 endif
+system-test-start:
+	CONTIV_NODES=3 vagrant up
```

Comment:
Makes sense. The reason I wanted to do this "3 nodes" only when system-test is running and for all other Makefile targets, this should not affect.